python/ray/monitor.py

```diff
@@ -216,6 +221,8 @@ def scan_db_client_table(self):
             return
 
         clients = self.state.client_table()
+        if type(clients) is list:
```

Comment:
note the line

```python
if self.use_raylet:
    return
```

which I added recently to avoid running this method for xray. We probably want to remove that.

Reply:
Ah, didn't see that. Removed this.